src/components/BannerApp/AppBanner.jsx

```diff
@@ -0,0 +1,16 @@
+import './AppBanner.css'
+import logoImage from '../../assets/images/logo.svg'
+export const AppBanner = () => {
+
+    return (
+        <div className='bannerAppContainer'>
+            <div className='imageContainer'>
```

Comment:
prepend the class name that you want with the file name. So If you have another component with an Image COntainer, the styles don't get mixed up. Name it like `appBannerImageContainer`

Reply:
Do the same with other class names